redeploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This is just a hack script and should be removed. Actual build and deploy should be like other images.